.github/workflows/community.yml

```diff
@@ -0,0 +1,32 @@
+on:
+  fork:
+  push:
+    branches: [ develop ]
+  issues:
+    types: [ opened ]
+  issue_comment:
+    types: [ created ]
+  pull_request_target:
+    types: [ opened ]
+  pull_request_review_comment:
+    types: [ created ]
+
+jobs:
+
+  welcome:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: ./src/welcome
```

Comment:
I'm not sure if this will work.
In the other repos the uses is `EddieJaoudeCommunity/gh-action-community/src/welcome@main`